definitions/json/EnglandWales Scrubbed/EnglandWales Scrubbed.json

```diff
@@ -719,6 +719,24 @@
     "ListElement": "Post",
     "DisplayOrder": 2
   },
+  {
+    "ID": "fl_HearingPanelPreference",
+    "ListElementCode": "No preference",
+    "ListElement": "No preference",
+    "DisplayOrder": 1
+  },
+  {
+    "ID": "fl_HearingPanelPreference",
+    "ListElementCode": "Judge",
+    "ListElement": "Judge",
+    "DisplayOrder": 2
+  },
+  {
+    "ID": "fl_HearingPanelPreference",
+    "ListElementCode": "Panel",
+    "ListElement": "Panel",
+    "DisplayOrder": 3
+  },
```

Comment:
Is it worth talking to Lahiru and putting the IC changes together on a branch. Lahiru has also added these values in albeit under a different ID. Rather than duplicating, is it worth having the work on a single branch?